runtime/runtime-params-estimator/src/utils.rs

```diff
@@ -223,3 +225,10 @@ pub(crate) fn generate_fn_name(index: usize, len: usize) -> Vec<u8> {
     }
     name
 }
+
+/// Create a WASM module that is empty except for a main method and a single data entry with n characters
+pub(crate) fn generate_data_only_contract(data_size: usize) -> Vec<u8> {
+    let payload = rand::thread_rng().sample_iter(&Alphanumeric).take(data_size).collect::<String>();
+    let wat_code = format!("(module (data \"{}\") (func (export \"main\")))", payload);
```

Comment:
```suggestion
    let wat_code = format!("(module (data \"{payload}\") (func (export \"main\")))");
```

the future is here :)